package.json

```diff
@@ -7,29 +7,39 @@
     "test": "test"
   },
   "engines": {
-    "node": ">=0.12.0"
+    "node": ">=4.0.0"
   },
   "dependencies": {
+    "bluebird": "^3.2.1",
     "pluralize": "1.2.x",
     "qs": "5.2.x",
     "request": "2.9.x",
     "singleton": "1.0.x"
   },
   "devDependencies": {
-    "coffee-script": "1.10.x",
-    "docco": "0.7.x",
+    "coveralls": "^2.11.6",
```

Comment:
I've missed those and I think it's appropriate to put coveralls into package.json instead of travis config file because it's nature to put the dependencies there but once again - personal preferences - you can adjust it as you want : )